website_wow/static/src/xml/snippet_options.xml

```diff
@@ -0,0 +1,537 @@
+                                </li>
+                                <li data-select_class="o_wow_animate zoomInLeft">
+                                    <a href="#">Zoom In Left</a>
+                                </li>
+                                <li data-select_class="o_wow_animate zoomInRight">
+                                    <a href="#">Zoom In Right</a>
+                                </li>
+                            </ul>
+                        </li>
+                        <li class="dropdown-submenu o_wow_type_zoom_exits">
+                            <a href="#">Zoom Exits</a>
+                            <ul class="dropdown-menu">
+                                <li data-select_class="o_wow_animate zoomOut">
+                                    <a href="#">Zoom Out</a>
+                                </li>
+                                <li data-select_class="o_wow_animate zoomOutUp">
+                                    <a href="#">Zoom Out Up</a>
+                                </li>
+                                <li data-select_class="o_wow_animate zoomOutDown">
+                                    <a href="#">Zoom Out Down</a>
+                                </li>
+                                <li data-select_class="o_wow_animate zoomOutLeft">
+                                    <a href="#">Zoom Out Left</a>
+                                </li>
+                                <li data-select_class="o_wow_animate zoomOutRight">
+                                    <a href="#">Zoom Out Right</a>
+                                </li>
+                            </ul>
+                        </li>
+                        <li class="dropdown-submenu o_wow_type_specials">
+                            <a href="#">Specials</a>
+                            <ul class="dropdown-menu">
+                                <li data-select_class="o_wow_animate hinge">
+                                    <a href="#">Hinge</a>
+                                </li>
+                                <li data-select_class="o_wow_animate jackInTheBox">
+                                    <a href="#">Jack-In-The-Box</a>
+                                </li>
+                                <li data-select_class="o_wow_animate rollIn">
+                                    <a href="#">Roll In</a>
+                                </li>
+                                <li data-select_class="o_wow_animate rollOut">
+                                    <a href="#">Roll Out</a>
+                                </li>
+                            </ul>
+                        </li>
+                    </ul>
+                </li>
+            </div>
+
+            <!-- Durations -->
+            <div data-js="o_wow_duration" t-att-data-selector="o_wow_selector">
+                  <li class="dropdown-submenu">
+                      <a href="#">
+                          <i class="fa fa-clock">
```

Comment:
Closing the `<a>` tag here instead of later (line 391) prevents the spacing issue above the first item in the sub-menu.

Also, `fa-clock` doesn't appear to be a valid font-awesome icon, but `fa-clock-o` is.

These things apply to the rest of your top-level menus too.
```
<a href="#"><i class="fa fa-clock-o" />Wow Duration</a>
```